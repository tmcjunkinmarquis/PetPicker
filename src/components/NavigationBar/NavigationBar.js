import React, { Component } from 'react';
import { Route, NavLink } from 'react-router-dom';
import Logo from '../Logo/Logo';

class NavigationBar extends Component {
  constructor(props) {
    super()
  }

  render() {
    return (
      <nav>
        <Logo />
        <NavLink to='/profile' className='nav'>Profile</NavLink>
        <NavLink to='/matches' className='nav'>Matches</NavLink>
        <NavLink to='/pets' className='nav'>Pets</NavLink>
        {/* <NavLink to='/signup' className='nav'>Sign Up</NavLink> */}
      </nav >
    )
  }
}

export default NavigationBar;