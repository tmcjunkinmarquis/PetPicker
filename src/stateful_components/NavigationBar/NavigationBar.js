import React, { Component } from 'react';

class NavigationBar extends Component {
  constructor (props){
    super()

  }

  render(){
    return(
      <div>
        <h3>I am the Navigation Bar</h3>
        
        <hr/>
      </div>
    )
  }
}

export default NavigationBar;