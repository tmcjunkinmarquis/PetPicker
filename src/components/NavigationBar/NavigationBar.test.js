/* eslint-disable max-len */

import React from 'react';
import { shallow } from 'enzyme';
import { NavigationBar } from './NavigationBar';

describe('NavigationBar', () => {
  let wrapper;
  let mockProps;

  beforeEach(() => {
    jest.resetAllMocks();
    mockProps = {};
    wrapper = shallow(<NavigationBar {...mockProps} />, {
      disableLifecycleMethods: true
    });
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
