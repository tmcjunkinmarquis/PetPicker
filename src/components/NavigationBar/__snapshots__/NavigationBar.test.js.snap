// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationBar matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavigationBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <nav>
        <div
                className="container"
        >
                <div
                        className="Nav-wrapper red lighten-1"
                >
                        <h3
                                className="left nav-title-pp"
                        >
                                Pet Picker
                        </h3>
                </div>
        </div>
</nav>,
      "className": "navbar-fixed ",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
>
          <div
                    className="Nav-wrapper red lighten-1"
          >
                    <h3
                              className="left nav-title-pp"
                    >
                              Pet Picker
                    </h3>
          </div>
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="Nav-wrapper red lighten-1"
>
            <h3
                        className="left nav-title-pp"
            >
                        Pet Picker
            </h3>
</div>,
          "className": "container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3
                className="left nav-title-pp"
>
                Pet Picker
</h3>,
              undefined,
            ],
            "className": "Nav-wrapper red lighten-1",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Pet Picker",
                "className": "left nav-title-pp",
              },
              "ref": null,
              "rendered": "Pet Picker",
              "type": "h3",
            },
            undefined,
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "nav",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <nav>
          <div
                    className="container"
          >
                    <div
                              className="Nav-wrapper red lighten-1"
                    >
                              <h3
                                        className="left nav-title-pp"
                              >
                                        Pet Picker
                              </h3>
                    </div>
          </div>
</nav>,
        "className": "navbar-fixed ",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="container"
>
            <div
                        className="Nav-wrapper red lighten-1"
            >
                        <h3
                                    className="left nav-title-pp"
                        >
                                    Pet Picker
                        </h3>
            </div>
</div>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="Nav-wrapper red lighten-1"
>
              <h3
                            className="left nav-title-pp"
              >
                            Pet Picker
              </h3>
</div>,
            "className": "container",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="left nav-title-pp"
>
                  Pet Picker
</h3>,
                undefined,
              ],
              "className": "Nav-wrapper red lighten-1",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Pet Picker",
                  "className": "left nav-title-pp",
                },
                "ref": null,
                "rendered": "Pet Picker",
                "type": "h3",
              },
              undefined,
            ],
            "type": "div",
          },
          "type": "div",
        },
        "type": "nav",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "disableLifecycleMethods": true,
    "hydrateIn": undefined,
  },
}
`;
